sea-orm-migration/src/migrator.rs

```diff
@@ -442,9 +442,37 @@ where
     C: ConnectionTrait,
 {
     match db.get_database_backend() {
-        DbBackend::MySql => MySql.query_tables(),
-        DbBackend::Postgres => Postgres.query_tables(),
-        DbBackend::Sqlite => Sqlite.query_tables(),
```

Comment:
would something like this work?

```rust
        #[cfg(feature = "sqlx-mysql")]
        DbBackend::MySql => MySql.query_tables(),
        #[cfg(feature = "sqlx-postgres")]
        DbBackend::Postgres => Postgres.query_tables(),
        #[cfg(feature = "sqlx-sqlite")]
        DbBackend::Sqlite => Sqlite.query_tables(),
```

if the feature is not enabled, the variant shouldn't exist

Reply:
Unfortunately not, as Rust needs the arms to be present. But the direction of though is right, thank you